core/backend/src/CloudSqlite.ts

```diff
@@ -37,8 +37,10 @@ export namespace CloudSqlite {
    * If the service is unavailable or returns an error, an empty token is returned.
    */
   export async function requestToken(args: RequestTokenArgs): Promise<AccessToken> {
-    // allow the userToken to be supplied via Rpc. If not supplied, or blank, use the backend's accessToken.
-    const userToken = args.userToken ? args.userToken : (await IModelHost.getAccessToken());
+    // allow the userToken to be supplied via args. If not supplied, or blank, use the backend's accessToken. If that fails, use the value from the current RPC request
+    let userToken = args.userToken ? args.userToken : await IModelHost.getAccessToken();
```

Comment:
This change allows CloudSqlite files to be opened by web backends using the credentials supplied by a single web client (vs. Ipc backends where the credentials are stored in IModelHost) to obtain an access token. The only previous use of this was for ViewStores, but workspaces also will need user credentials. See corresponding change in IModelReadRpcImpl.ts.

(Note: GCS workspaces are public, hence didn't need an access token)

Reply:
@pmconne feel free to resolve these comments whenever you like